Tests/CacheAdvanceTests/SQLitePerformanceComparisonTests.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  Created by Dan Federman on 4/23/20.
+//  Copyright © 2020 Dan Federman.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS"BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SQLite3
+import XCTest
+
+final class SQLitePerformanceComparisonTests: XCTestCase {
+
+    // MARK: XCTestCase
+
+    override func tearDown() {
+        sqlite3_close(db)
+
+        super.tearDown()
+    }
+
+    // MARK: Behavior Tests
+
+    func test_performance_append_sqlite() throws {
+        measure {
+            createDatabase()
```

Comment:
The below tests doesn't include DB creation time. This test does.
What's a bit weirder too is that when you create the DB you delete the old DB.
So if this test ran second, it would increase in time since it would need to delete the DB from the previous test. But if it's first there is no DB to delete.
Similar to my comments on the other tests I wonder if we should pull DB creation of the `measure` block.
We could even have a test for "starting up" each cache if we wanted to measure that separately.

Reply:
Good call-out. These tests are now a bit more apples-to-apples.